packages/dart/noports_core/lib/src/sshnpd/sshnpd_impl.dart

```diff
@@ -326,13 +370,6 @@ class SshnpdImpl implements Sshnpd {
   }
 
   Future<void> _handlePublicKeyNotification(AtNotification notification) async {
-    if (!isFromAuthorizedAtsign(notification)) {
-      logger.shout('Notification ignored from ${notification.from}'
-          ' which is not in authorized list $managerAtsigns.'
-          ' Notification value was ${notification.value}');
-      return;
-    }
-
```

Comment:
Code block unnecessary, as the guard is in _notificationHandler which calls this method). Likewise below in other methods